woocommerce/woocommerce
# Improve experimental select control accessibility

### All Submissions:






### Changes proposed in this Pull Request:

Improves the new `SelectControl` component accessibility by
* Announcing when items are selected
* Announcing when an item is removed with the correct item string
* Announcing when items are loaded in the async example
* Announcing when items are selected via arrow keys

### How to test the changes in this Pull Request:

1. Run `npm run test`
2. Go through the storybook examples under `experimental/SelectControl`
3. Turn on VoiceOver and open Safari
4. Check the list of the above accessibility improvements

See https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-select-only.html for correct interactions.

### Other information:




### FOR PR REVIEWER ONLY:
